atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/cleaner/PuncherStore.java

```diff
@@ -44,9 +44,4 @@ default boolean isInitialized() {
      * Find the latest timestamp created at or before timeMillis.
      */
     Long get(Long timeMillis);
-
-    /**
-     * Get the time in millis for the greatest timestamp punched less than or equal to the given timestamp.
-     */
-    long getMillisForTimestamp(long timestamp);
```

Comment:
We've got at least one internal use of this that we'll have to resolve if we want to remove it.

Reply:
won't remove